raiden/tests/integration/transfer/test_refundtransfer.py

```diff
@@ -457,15 +458,16 @@ def patched_update(raiden, event):
         retry_timeout=app0.raiden.alarm.sleep_time,
     )
 
-    with gevent.Timeout(10):
+    timeout = 30 if blockchain_type == 'parity' else 10
+    with gevent.Timeout(timeout):
         unlock_app0 = wait_for_state_change(
             app0.raiden,
             ContractReceiveChannelBatchUnlock,
             {'participant': app0.raiden.address},
             retry_timeout,
         )
     assert unlock_app0.returned_tokens == 50
-    with gevent.Timeout(10):
+    with gevent.Timeout(timeout):
```

Comment:
couldn't you re-use `timeout(blockchain_type)` as above?

Reply:
I think it's more readable this way, than calling a function in a different module. I thought about parametrizing `timeout(blockchain_type)` for different timeouts, but the 10 second case doesn't happen that often.